geth/node/node.go

```diff
@@ -23,6 +23,7 @@ import (
 	whisper "github.com/ethereum/go-ethereum/whisper/whisperv5"
 	"github.com/status-im/status-go/geth/log"
 	"github.com/status-im/status-go/geth/params"
+	whisperMetrics "github.com/status-im/status-go/metrics/whisper"
```

Comment:
very uncommon to use camel case for package names, can it be just `whispermetricss`?

Reply:
I have actually seen it a few times :D but I think I would lean towards `whispermetricss` or even `wmetrics` :)